social/twitter/27-twitter.js

```diff
@@ -631,31 +848,31 @@ module.exports = function(RED) {
                         } else {
                             mediaPromise = Promise.resolve();
                         }
-                        mediaPromise.then(function(mediaId) {
+                        mediaPromise.then(function (mediaId) {
                             var params = msg.params || {};
                             params.status = msg.payload;
                             if (mediaId) {
                                 params.media_ids = mediaId;
                             }
-                            node.twitterConfig.post("https://api.twitter.com/1.1/statuses/update.json",{},params).then(function(result) {
+                            node.twitterConfig.post("https://api.twitter.com/1.1/statuses/update.json", {}, params).then(function (result) {
```

Comment:
So there is this mix of twitter 1.1 for the Twitter Out node, and 2 for the Twitter in node.  It would be fantastic to have them both on Twitter 2.  At the moment if I use this branch and deploy first a twitter in, and then a twitter out node I get an exception I think because the two configurations of credentials clash `24 Oct 22:45:40 - [info] Updated flows
24 Oct 22:45:40 - [red] Uncaught Exception:
24 Oct 22:45:40 - [error] Error: aborted
    at connResetException (node:internal/errors:705:14)
    at TLSSocket.socketCloseListener (node:_http_client:454:19)
    at TLSSocket.emit (node:events:525:35)
    at node:net:301:12
    at TCP.done (node:_tls_wrap:588:7)`